packages/netdog/netdog.spec

```
@@ -0,0 +1,85 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}netdog
+Version: 0.1.1
+Release: 0%{?dist}
+Summary: Bottlerocket network configuration helper
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source2: run-netdog.mount
+Source3: write-network-status.service
+Source4: netdog-tmpfiles.conf
+Source6: 00-resolved.conf
+Source7: generate-network-config.service
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}netdog
+
+%description
+%{summary}.
+
+%package -n %{_cross_os}netdog-systemd-networkd
+Summary: Bottlerocket network configuration helper
+Provides: %{_cross_os}netdog = 2:
+Requires: %{_cross_os}systemd-networkd
+Requires: %{_cross_os}systemd-resolved
+%description -n %{_cross_os}netdog-systemd-networkd
+%{summary}.
+
+%package -n %{_cross_os}netdog-wicked
+Summary: Bottlerocket network configuration helper
+Provides: %{_cross_os}netdog = 1:
+Requires: %{_cross_os}wicked
+%description -n %{_cross_os}netdog-wicked
+%{summary}.
+
+%prep
+%setup -T -c
+%cargo_prep
+
+%build
+mkdir bin
+
+echo "** Build Netdog Binaries"
+%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
+    -p netdog \
+    --target-dir=${HOME}/.cache/networkd
+%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
+    -p netdog \
+    --features wicked \
+    --target-dir=${HOME}/.cache/wicked
```

I'd like to be explicit in the first branch here and not default to one feature or the other:
```suggestion
%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
    -p netdog \
    --features networkd \
    --target-dir=${HOME}/.cache/networkd
%cargo_build --manifest-path %{_builddir}/sources/Cargo.toml \
    -p netdog \
    --features wicked \
    --target-dir=${HOME}/.cache/wicked
```

---

So you would like me to return the netdog package to having two features instead of one, Matt's main argument for having the one is that --all-features would still work (albeit will only test the wicked branch)

---

I have a preference for non-mutually-exclusive features (the way you have it now), but perhaps you can use `--features default` or `--default-features=true` to make this more self documenting.